### PR TITLE
Db proverbs

### PR DESCRIPTION
**Pull request by Joanartdacosta**
- load info about proverbs from mongodb
- adapt UI components to info from mongodb

### CODE REVIEW

File: app/helpers/api-util.js [hunk 1]
```diff
@@ -167,26 +167,27 @@ export async function getByIdPassport(id) {
...
 
-export async function getAllProverbs() {
+export async function getProverbs() {
   try {
     const client = await clientPromise;
     const db = client.db("roads");
 
-    const allProverbs = await db.collection("proverbs").find({}).toArray();
+    const currentDate = new Date();
+    let currentMonth = currentDate.getMonth() + 1;
```
Comment (doublethink13) [inline app/helpers/api-util.js:176]: const currentMonth = new Date().getMonth() + 1;

File: app/helpers/api-util.js [hunk 2]
```diff
@@ -167,26 +167,27 @@ export async function getByIdPassport(id) {
...
 
-    return allProverbs.map((mongoDbItem) => {
+    const proverbs = await db
+      .collection("proverbs")
+      .find({ id: currentMonth })
```
Comment (doublethink13) [inline app/helpers/api-util.js:180]: id implies this is something unique but we are using currentMonth which does not seem like something unique

the proverb id should be unique

the proverb document can have a field called month that you use to filter

Comment (Joanartdacosta) [reply app/helpers/api-util.js:180]: There is not two months that are the same, that's why I used the id to identify the month

File: app/helpers/api-util.js (hunk 1 above)
Comment (Joanartdacosta) [reply app/helpers/api-util.js:176]: well noted, thank you so much!

File: app/helpers/api-util.js (hunk 2 above)
Comment (doublethink13) [reply app/helpers/api-util.js]: but maybe there will be proverbs that have the same month?

id != month

your attribute should be informative about the information they carry

an id implies the value is unique across all proverbs

month would imply the month where the proverb is used

you should also use findOne together with the month attribute

Comment (Joanartdacosta) [reply app/helpers/api-util.js]: Indeed! Changes done, according your suggestions

**Pull request merged**